conda.recipe/meta.yaml

```diff
@@ -11,11 +11,7 @@ build:
   # so we have to exempt them from the list of initial files
   # for conda-build to realize they should be included.
   always_include_files:
-    - bin/activate
-    - bin/deactivate
-    - bin/conda
```

Comment:
@groutr Don't remove these.  In fact there should be a selector for `unix`, and then other files for `win`